packages/e2e-tests/specs/experiments/comments-query.test.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * WordPress dependencies
+ */
+import { insertBlock, setOption } from '@wordpress/e2e-test-utils';
+/**
+ * WordPress dependencies
+ */
+
+// It might make sense to include a similar test in WP core (or move this one over).
+// See discussion here: https://github.com/WordPress/gutenberg/pull/32797#issuecomment-864192088.
+describe( 'Comment Query Loop', () => {
+	it( 'Comment Query block insertion', async () => {
+		// We won't select the option that we updated and will also remove some
+		// _transient options that seem to change at every update.
+		setOption( 'page_comments', true, 'options-discussion.php' );
```

Comment:
```suggestion
		await setOption( 'page_comments', true, 'options-discussion.php' );
```

Otherwise, it fails 😬

Reply:
Also, let's put this into a `beforeAll`, with a matching teardown in `afterAll`. Doing this kind of setup/teardown in those routines tends to be much more stable then if done in individual tests, as our Jest setup adds some other magic to those...